store/src/java/com/zimbra/cs/service/admin/AdminDocumentHandler.java

```diff
@@ -588,10 +588,8 @@ protected Element proxyIfNecessary(Element request, Map<String, Object> context)
             if (acctId != null) {
                 Account acct = getAccount(prov, AccountBy.id, acctId, zsc.getAuthToken());
                 if (acct != null && !Provisioning.onLocalServer(acct, reasons)) {
-                    if (zsc.getHopCount() > 2 || (ZimbraLog.soap.isDebugEnabled())) {
```

Comment:
why the hopCount check is removed?

Reply:
The original code was intended to shout out when something odd was happening but the point of this Jira ticket is that proxying event are always useful to know about, even when it is normal behavior.  Makes it easier to understand what is happening on the server.